Source/LuaBridge/Array.h

```diff
@@ -0,0 +1,54 @@
+// https://github.com/vinniefalco/LuaBridge
+//
+// Copyright 2018, Dmitry Tarakanov
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <LuaBridge/detail/Stack.h>
+
+#include <array>
+
+namespace luabridge {
+
+template <class T, std::size_t s>
+struct Stack <std::array <T,s> >
```

Comment:
The coding style has been changed recently.
Now there's `.clang-format` file, so if you use MSVS, you can simply `Ctrl+K, Ctrl+D`, otherwise you can use the `clang-format` tool in order to format the file.